compiler-cli/src/main.rs

```diff
@@ -93,6 +93,14 @@ use clap::{
 };
 use strum::VariantNames;
 
+#[derive(Args, Debug, Clone)]
+struct UpdateOptions {
+    /// (optional) Names of the packages to update
+    /// If omitted, all dependencies will be updated
+    #[arg(verbatim_doc_comment)]
```

Comment:
What does this mean?

Reply:
If you are asking about the `verbatim_doc_comment` - this tells `clap-rs` to use the comment verbatim, ie. respect the newline between the two comment lines. I thought this was better visually, and emphasized the default a bit better than one long line.

A visual:
```
Arguments:
  [PACKAGES]...  (optional) Names of the packages to update
                 If omitted, all dependencies will be updated
```
versus:
```
Arguments:
  [PACKAGES]...  (optional) Names of the packages to update. If omitted, all dependencies will be updated.
```

`clap-rs` docs regarding preprocessing: https://docs.rs/clap/latest/clap/_derive/index.html#pre-processing 

There are two existing uses of `verbatim_doc_comment` in the codebase: https://github.com/search?q=repo%3Agleam-lang%2Fgleam%20verbatim_doc_comment&type=code

I can rewrite the comment if you'd prefer it to be a single line. It's also possible I overlooked a more elegant way to communicate this to users?

Reply:
That's great, thank you for the explanation!